src/main/java/io/github/thebusybiscuit/slimefun4/core/multiblocks/MultiBlockMachine.java

```diff
@@ -104,12 +104,16 @@ public void postRegister() {
     @Override
     public void load() {
         super.load();
-
-        for (ItemStack recipeItem : displayRecipes) {
-            SlimefunItem item = SlimefunItem.getByItem(recipeItem);
-
-            if (item == null || !item.isDisabled()) {
-                recipes.add(new ItemStack[] { recipeItem });
+        
+        for (int i = 0; i < displayRecipes.size(); i += 2) {
```

Comment:
You should also check if the size of this list is a multiple of 2 before.

Reply:
why? (Sorry if that came out blatant just genuinely confused)

Reply:
Adding a case for if the list is not a multiple of 2.